notes/noteModel.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const NoteSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: true,
+    },
+    textBody: {
+        type: String,
+        required: true,
+    },
+    tags: [String],
```

Comment:
Will you be keeping a list of notes on the User object, then? Or will you add a field here to keep track of which user the note belongs to? Either one is totally valid 👍